Automattic/wp-calypso
# Full Site Editing: remove blank theme

#### Changes proposed in this Pull Request

With new rendering approach blank theme is no longer needed for FSE plugin to work.

#### Testing instructions

1. Verify that all of the theme references have been removed from FSE plugin.
2. Run `npx lerna run dev --scope='@automattic/full-site-editing'` from Calypso root folder and verify that everything still works as expected.
3. Run `npx lerna run build --scope='@automattic/full-site-editing' ` from Calypso root folder and verify that everything still works as expected.
4. Navigate to `wp-admin/themes.php` of your test site and verify that Blank theme is no longer present.
5. Check generated CircleCI artifacts of this PR and make sure that there is no Blank theme folder there anymore.

Fixes https://github.com/Automattic/wp-calypso/issues/34303
